package/gluon-wlan-encryption-psk/check_site.lua

```diff
@@ -0,0 +1 @@
+need_table({'wlan_encryption_psk'})
```

Comment:
According to the [comment of CodeFetch](https://github.com/freifunk-gluon/gluon/pull/1719/files#r324689064) the min and may length should be checked by check_site.lua